logging/clusterlogging/nodeSelector.yaml

```diff
@@ -12,14 +12,10 @@ spec:
         es: deploy
       nodeCount: 1
       resources:
-        limits:
-          memory: 2Gi
         requests:
           cpu: 200m
           memory: 2Gi
-      storage:
-        size: "20G"
-        storageClassName: "gp2"
+      storage: {}
```

Comment:
Storage size removed looks like a mistake. I think only the line with storageClassName should be removed.

Reply:
@liangxia I don't want to enable storage, so I change it to empty.
```
$ oc get clusterlogging -oyaml
apiVersion: v1
items:
- apiVersion: logging.openshift.io/v1
  kind: ClusterLogging
  metadata:
    creationTimestamp: "2019-12-25T01:57:21Z"
    generation: 1
    name: instance
    namespace: openshift-logging
    resourceVersion: "282828"
    selfLink: /apis/logging.openshift.io/v1/namespaces/openshift-logging/clusterloggings/instance
    uid: 9809753b-85a0-402f-9d66-72ca9a2979cc
  spec:
    collection:
      logs:
        fluentd: {}
        type: fluentd
    curation:
      curator:
        schedule: '*/10 * * * *'
      type: curator
    logStore:
      elasticsearch:
        nodeCount: 1
        redundancyPolicy: ZeroRedundancy
        resources:
          limits:
            memory: 2Gi
          requests:
            cpu: 200m
            memory: 2Gi
        storage: {}
      type: elasticsearch
    managementState: Managed
```

Reply:
Ack as you don't want to enable storage.